exo-add/features/without-arguments.feature

```diff
@@ -9,14 +9,14 @@ Feature: interactive scaffolding
     Given I am in the root directory of an empty application called "test app"
     When starting "exo-add service" in this application's directory
     And entering into the wizard:
-      | FIELD                         | INPUT   |
-      | Role of the service to create | web     |
-      | Type of the service to create | web     |
-      | Description                   | testing |
-      | Author                        | tester  |
-      | Template                      |         |
-      | Name of the data model        | web     |
-      | Protection level              | public  |
+      | FIELD                         | INPUT          |
+      | Role of the service to create | web            |
+      | Type of the service to create | web            |
+      | Description                   | testing        |
+      | Author                        | tester         |
+      | Template                      | exoservice-es6 |
```

Comment:
Does this work? Afaik there is no way to select things in the wizard, because one would have to hit the _down_ cursor key a number of times and then hit Enter.